dsc/examples/wmi_inventory.dsc.yaml

```diff
@@ -0,0 +1,42 @@
+$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
+resources:
+- name: WMI
+  type: Microsoft.Windows/WMI
+  properties:
+    resources:
+      - name: computer system
+        type: root.cimv2/Win32_ComputerSystem
+        properties:
+          name:
+          domain:
+          totalphysicalmemory:
+          model:
+          manufacturer:
+      - name: operating system
+        type: root.cimv2/Win32_OperatingSystem
+        properties:
+          caption:
+          version:
+          osarchitecture:
+          oslanguage:
+      - name: system enclosure
+        type: root.cimv2/Win32_SystemEnclosure
+        properties:
+          manufacturer:
+          model:
+          serialnumber:
+      - name: bios
+        type: root.cimv2/Win32_BIOS
+        properties:
+          manufacturer:
+          version:
+          serialnumber:
+      - name: network adapter
+        type: root.cimv2/Win32_NetworkAdapter
+        properties:
+          name:
+          macaddress:
+          adaptertype:
+          netconnectionid:
+          serviceName:
+          netconnectionstatus: 2
```

Comment:
Is my understanding correct that this is the only instance in this file that will use `WHERE` clause?

Reply:
Yes